projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1861,8 +1862,16 @@ public static function wrap_message_in_html_tags( $title, $body, $footer ) {
 			return $body;
 		}
 
-		$template = '';
-		$style    = '';
+		$template       = '';
+		$style          = '';
+		$event          = new Jetpack_Tracks_Event(
+			(object) array(
+				'_en' => 'jetpack_forms_email_open',
+				'_ui' => hash_hmac( 'md5', get_option( 'admin_email' ), JETPACK__VERSION ),
+				'_ut' => 'anon',
+			)
+		);
+		$tracking_pixel = '<img src="' . $event->build_pixel_url() . '" />';
```

Comment:
Can you add `alt="" width="1" height="1"` to ensure the tracking pixel image stays at 1px width/height even when it doesn't load due to DNS or some plugin blocking it from loading?

I fixed a similar issue in another email some time ago:

![image](https://github.com/user-attachments/assets/7209a213-51ca-4acd-adbf-b7f71af2e6f0)